src/app/pages/homepage/homepage.ts

```diff
@@ -4,6 +4,7 @@ import {MatButtonModule} from '@angular/material/button';
 import {FooterModule} from '../../shared/footer/footer';
 import {RouterModule} from '@angular/router';
 import {ComponentPageTitle} from '../page-title/page-title';
+import { LicenseModule } from '../license/license';
```

Comment:
Please remove the spaces surrounding the import.

```suggestion
import {LicenseModule} from '../license/license';
```